starlette/middleware/wsgi.py

```diff
@@ -93,7 +94,7 @@ def __init__(self, app: typing.Callable, scope: Scope) -> None:
                     self.exc_info[1], self.exc_info[2]
                 )
         finally:
-            if not sender.done():
```

Comment:
I don't see any value in the two lines changed here.

Reply:
`self.loop.create_task()` may raise an exception, then `sender` will be referenced before assignment.

Reply:
Okay, then let's move `sender = ` to just outside the `try ... finally` block